elftools/elf/structs.py

```diff
@@ -153,6 +154,16 @@ def _create_shdr(self):
             self.Elf_xword('sh_entsize'),
         )
 
+    def _create_chdr(self):
+        fields = [
```

Comment:
Ok, I also found this: <https://docs.oracle.com/cd/E53394_01/html/E54813/section_compression.html> I see this documentation is already referenced in `structs.py:_create_gnu_verneed`, although it seems that the section numbering scheme have changed: the Object File Format is now Chapter 13 whereas it used to be Chapter 7. I’ll add a commit to update these.